src/peer-store/persistent/index.js

```diff
@@ -0,0 +1,225 @@
+'use strict'
+
+const debug = require('debug')
+const log = debug('libp2p:persistent-peer-store')
+log.error = debug('libp2p:persistent-peer-store:error')
+
+const { Key } = require('interface-datastore')
+const multiaddr = require('multiaddr')
+const PeerId = require('peer-id')
+
+const PeerStore = require('..')
+
+const {
+  ADDRESS_NAMESPACE,
+  COMMON_NAMESPACE,
+  PROTOCOL_NAMESPACE
```

Comment:
nit: consider calling these `NAMESPACE_<>`, it can make the values easier to auto suggest in IDEs